pymc3/sampling.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict, List, Any, Optional # pylint: disable=unused-import
```

Comment:
If you remove any from this import, does pylint complain? We prefer to not ignore lint